receiver/hostmetricsreceiver/internal/scraper/networkscraper/network_scraper.go

```diff
@@ -45,11 +46,12 @@ type scraper struct {
 	bootTime    func() (uint64, error)
 	ioCounters  func(bool) ([]net.IOCountersStat, error)
 	connections func(string) ([]net.ConnectionStat, error)
+	conntrack   func() ([]net.FilterStat, error)
 }
 
 // newNetworkScraper creates a set of Network related metrics
 func newNetworkScraper(_ context.Context, cfg *Config) (*scraper, error) {
-	scraper := &scraper{config: cfg, bootTime: host.BootTime, ioCounters: net.IOCounters, connections: net.Connections}
+	scraper := &scraper{config: cfg, bootTime: host.BootTime, ioCounters: net.IOCounters, connections: net.Connections, conntrack: net.FilterCounters}
```

Comment:
Should we use ` FilterCountersWithContext` instead, so that we can pass the context in case we want to have cancellation at some point? 

Not super convinced that we should do this here (`IOCounters` also has `IOCountersWithContext` and we are not using it), so maybe you can just open an issue and we can deal with it in a separate PR.